provisioning/service/src/Config/IndividualEnrollment.cs

```diff
@@ -201,46 +198,13 @@ public override string ToString()
         /// </remarks>
         /// <exception cref="ArgumentException">if the provided string does not fit the registration Id requirements</exception>
         [JsonProperty(PropertyName = "registrationId")]
-        public string RegistrationId
-        {
-            get
-            {
-                return _registrationId;
-            }
-
-            private set
-            {
-                _registrationId = value;
-            }
-        }
-
-        private string _registrationId;
+        public string RegistrationId { get; private set; }
 
         /// <summary>
         /// Desired IoT Hub device Id (optional).
         /// </summary>
         [JsonProperty(PropertyName = "deviceId", DefaultValueHandling = DefaultValueHandling.Ignore)]
-        public string DeviceId
-        {
-            get
-            {
-                return _deviceId;
-            }
-
-            set
-            {
-                if (value == null)
```

Comment:
Not sure what this logic is. If it is null, then assign null, otherwise assign the value? Isn't that just the same as assigning the value?

With this custom logic removed, we can use an autoproperty.